packages/get-uri/test/util.ts

```diff
@@ -5,5 +5,5 @@ export async function toBuffer(stream: Readable): Promise<Buffer> {
 	for await (const chunk of stream) {
 		chunks.push(chunk);
 	}
-	return Buffer.concat(chunks);
+	return Buffer.concat(chunks.map((chunk) => new Uint8Array(chunk)));
```

Comment:
Curious - why was this necessary?

Reply:
I had a TypeError locally, but I believe I'm using a different version of TypeScript so have removed this change.